FileMapper/fileUploadForm_script.html

```diff
@@ -0,0 +1,257 @@
+      // and on success of that function executes the updateUrl function
+      google.script.run
+          .withSuccessHandler(function(url) {
+            document.getElementById('submitButton').value = "Submit";
+            showMessage(
+                "<strong>Successfully</strong> uploaded the file <strong> " +
+                filename +
+                "</strong> to Google Drive and mapped it to the local path <strong>" +
+                localPathValue +
+                "</strong>. <br> To check it out visit this <strong><a href='" +
+                url + "' style='color: inherit;'>Drive Link</a></strong> !!");
+          })
+          .withFailureHandler(function() {
+            document.getElementById('submitButton').value = "Submit";
+            showMessage(
+                "Failed to upload the file to Google Drive and map it to the local path <strong>" +
+                localPathValue + "</strong>.<br> Please try again.");
+          })
+          .processForm(data, mimetype, filename, localPathValue,
+                      document.getElementById('hiddenId').innerHTML);
+    } 
+    fr.readAsDataURL(file);
+  }
+
+  /**
+   * Get the extension from the file path
+   *
+   * @param {String} path The local path
+   * @return {String} extension The extension of the local path
+   */
+  function getExtensionFromLocalPath(path) {
+    var index = path.lastIndexOf(".");
+    var extension = "";
+    if (index != -1) {
+      extension = path.substr(index + 1);
+    }
+
+    // To emulate Windows behavior
+    // In windows, files without extension are text files
+    return (extension === "") ? "txt" : extension;
+  }
+
+  /**
+   * Function to return the MimeType based on the file extension
+   *
+   * @param {String} extension The extension of the local path
+   * @return {String} MimeType of the file based on it's extension
+   */
+  function getMimetypeFromExtension(extension) {
+    if (extension === "xls" || extension === "xlsx" || extension === "csv")
+      return 'GOOGLE_SHEETS';
+
+    if (extension === "doc" || extension === "docx" || extension === "pdf" ||
+        extension === "txt")
+      return 'GOOGLE_DOCS';
+
+    if (extension === "ppt" || extension === "pptx")
+      return 'GOOGLE_SLIDES';
+
+    return 'UNKNOWN';
+  }
+
+  /**
+   * Function to check if the mimetype of the local path file and the uploaded
+   * file matches or not
+   *
+   * @param {String} uploadedFile File name of the uploaded file
+   */
+  function checkIfExtensionMatches(uploadedFile) {
+    var localPath = document.getElementById("localpathinput").value;
+
+    var uploadedFileExtension = getExtensionFromLocalPath(uploadedFile);
+    var localPathExtension = getExtensionFromLocalPath(localPath);
+
+    if (getMimetypeFromExtension(localPathExtension) ===
+        getMimetypeFromExtension(uploadedFileExtension)) {
+      showUploadedFileMessage("File type matches to the local file type", true);
```

Comment:
to the -> the, also in the message below

Reply:
Fixed